patroni/postgresql/mpp/__init__.py

```diff
@@ -93,34 +93,51 @@ def __init__(self, postgresql: 'Postgresql', config: Dict[str, Union[str, int]])
 
     @abc.abstractmethod
     def handle_event(self, cluster: Cluster, event: Dict[str, Any]) -> None:
-        pass
+        """Handle event send from worker node.
+
+        :param cluster: object containing stateful information for the cluster.
+
+        :param config: dictionary of events to be handled
+        """
 
     @abc.abstractmethod
     def sync_pg_dist_node(self, cluster: Cluster) -> None:
-        pass
+        """Sync meta data to coordinator.
```

Comment:
```suggestion
        """Sync meta data on the coordinator.
```

And maybe the method should be called differently.